samples/spring-valueentity-shopping-cart/src/main/java/com/example/ShoppingCartController.java

```diff
@@ -0,0 +1,106 @@
+package com.example;
+
+import com.example.api.ShoppingCartDTO;
+import com.example.domain.ShoppingCart;
+import com.google.protobuf.any.Any;
+import com.google.protobuf.Empty;
+import kalix.javasdk.DeferredCall;
+import kalix.javasdk.action.Action;
+import kalix.javasdk.action.ActionCreationContext;
+import kalix.springsdk.KalixClient;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletionStage;
+
+@RequestMapping("/carts")
+public class ShoppingCartController extends Action {
```

Comment:
Shouldn't this be a part of the api package?

Reply:
it should, tks 🙇